e2e/cypress/integration/messaging/message_reply_part2_spec.js

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+const TIMEOUTS = require('../../fixtures/timeouts');
+
+describe('Message Reply', () => {
+    let mainChannel;
+    let otherChannel;
+    let rootId;
+
+    before(() => {
+        // # Create main channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            mainChannel = channel;
+
+            // # Create other channel
+            cy.apiCreateChannel(team.id, 'other', 'other').then(({newChannel}) => {
```

Comment:
MM-T2136 is failing on me. I believe this should be,
```suggestion
            cy.apiCreateChannel(team.id, 'other', 'other').then(({channel: newChannel}) => {
```

Reply:
oh yeah, didn't notice it was destructuring the object, thanks :+1: